src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method `getStatistic` is quite large and handles multiple responsibilities such as reading from a file, processing data, and writing to a file. Consider dividing it into smaller methods to improve readability and maintainability, as per the checklist item 'Divide program logic'.